assignments/arrays.js

```diff
@@ -63,32 +63,54 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log("Car 33 is a " + inventory[32].car_year + " " + inventory[32].car_make + " " + inventory[32].car_model);
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+console.log(inventory[inventory.length - 1].car_make + " " + inventory[inventory.length - 1].car_model);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+for(var i = 0; i < inventory.length; i++){
+    var models = inventory[i].car_model;
+    carModels.push(models);
+    carModels.sort();
+}
```

Comment:
here I would recommend you to use the `.sort()` method after the for loop because since its inside you are sorting through the array every instance of the loop so 50 times instead of 1 if you would do .sort() outside of the loop